SPECS/vim/vim.spec

```
@@ -30,7 +31,7 @@ The vim extra package contains a extra files for powerful text editor.
 %prep
 %autosetup -p1
 echo '#define SYS_VIMRC_FILE "%{_sysconfdir}/vimrc"' >> src/feature.h
-%py3_shebang_fix runtime/tools/demoserver.py
+%{py3_shebang_fix} runtime/tools/demoserver.py
```

I think this will make the macro run without any arguments and fail. Please double-check, though.
```suggestion
%py3_shebang_fix runtime/tools/demoserver.py
```